asabiris/orchestration/sendmail.py

```diff
@@ -119,6 +125,9 @@ async def render(self, template, params):
 
 		jinja_output will be used for extracting subject.
 		"""
+		if template.startswith("/"):
```

Comment:
My testing library does not contain "/Templates" directory at all and the e-mail still proceeds.

```python
if template.startswith("/"):
    assert template.startswith("/Templates"), "Your template must be stored in /Templates directory"
else:
    template = "/Templates/" + template
```

@ateska This is what we agreed on during review. However, as I saw during demo, further breakdown of the Templates folder (Mails, Slack etc...) is being prepared together with asab-library, right?
It only makes my feeling about the necessity of using absolute paths stronger.
Just now, I cannot really decide, what the else branch should look like.

Reply:
@eliska-n Yes , because iris accepts relative paths.